pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc

```diff
@@ -0,0 +1,18 @@
+= Policy Development Kit Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Policies enforce regulations to help manage security, control traffic, and improve API adoption. Flex Gateway provides many xref:gateway::policies-included-directory.adoc[included policies], but they might not cover all your specific business needs.
+
+The Policy Development Kit (PDK) is a software development kit that enables Flex Gateway users to develop custom policies with ease.
+
+PDK provides step-by-step instructions to guide you through the policy development lifecycle, from implementing your Rust source code to releasing your policy on Exchange. Like included policies, you can apply custom policies to Flex Gateway by using API Manager or YAML configuration files.
+
+PDK also provides template policies to help you get started with policy development.
+
+== Before You Begin
```

Comment:
This section heading is usually reserved for tutorial docs, not overviews. I think these two links could be under a "See Also" section.

Reply:
https://github.com/mulesoft/docs-gateway/pull/504